docs/synapse/httpapi.rst

```diff
@@ -422,9 +422,10 @@ For more information about Storm APIs, including opts behavior, see :ref:`dev_st
         .. code:: python3
             :name: aiohttp api/v1/storm example
 
-            import json
             import pprint
 
+            import synapse.lib.json as s_json
```

Comment:
These examples do not assume that synapse is available, we should be able to continue to use `json` from the standard library in them.